api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -296,7 +296,19 @@ message ExtAuthz {
   // Additional metadata to be added to the filter state for logging purposes. The metadata will be
   // added to StreamInfo's filter state under the namespace corresponding to the ext_authz filter
   // name.
+  //
+  // If this is unset and ``emit_filter_state_stats`` is false, the filter will not emit any filter
```

Comment:
What happens if this field is set and `emit_filter_state_stats` is true?  Won't that cause a conflict?

Reply:
No, ext_authz would emit an ExtAuthzLoggingInfo object with stats but the filter_metadata field would be nullopt.